server/chat-plugins/wifi.tsx

```diff
@@ -1374,317 +1442,454 @@ function makePageHeader(user: User, pageid?: string) {
 		submitted: `View Submitted`,
 		'submitted-add': `Submit`,
 	};
-	const icons: Record<keyof typeof titles, string> = {
-		create: `<i class="fa fa-sticky-note"></i>`,
-		stored: `<i class="fa fa-paste"></i>`,
-		'stored-add': `<i class="fa fa-paste"></i>`,
-		submitted: `<i class="fa fa-inbox"></i>`,
-		'submitted-add': `<i class="fa fa-inbox"></i>`,
+	const icons: Record<keyof typeof titles, Chat.VNode> = {
+		create: <i className="fa fa-sticky-note"></i>,
+		stored: <i className="fa fa-paste"></i>,
+		'stored-add': <i className="fa fa-paste"></i>,
+		submitted: <i className="fa fa-inbox"></i>,
+		'submitted-add': <i className="fa fa-inbox"></i>,
 	};
-	let buf = `<button class="button" style="float:right" name="send" value="/j view-giveaways${pageid?.trim() ? `-${pageid.trim()}` : ''}"><i class="fa fa-refresh"></i> Refresh</button>`;
-	buf += `<h1>Wi-Fi Giveaways</h1>`;
+	const buf = [];
+	buf.push(<button className="button" style={{float: 'right'}} name="send" value={
```

Comment:
Can you fix all the other `className`s too? They should all be `class`.

Reply:
Yes